src/org/sosy_lab/cpachecker/cpa/smg/graphs/SMGPredicateRelation.java

```diff
@@ -149,6 +149,30 @@ public void addExplicitRelation(
     }
   }
 
+  public SymbolicRelation findAnySymbolicRelation(SMGValue pValue) {
+    if (!smgValuesDependency.get(pValue).isEmpty()) {
```

Comment:
Please introduce variable for smgValuesDependency.get(pValue) to reduce number of querying